os/drivers/audio/alc5658.c

```diff
@@ -726,7 +726,6 @@ static void alc5658_io_err_cb(FAR struct i2s_dev_s *dev, FAR void *arg, int flag
 	/* Call upper callback, let it post msg to user q
 	 * apb is set NULL, okay? Rethink
 	*/
-	lldbg("alc5658_io_err_cb flags: 0x%x\n", flags);
```

Comment:
I think you can add this info audio verbose debug.

Reply:
No. it gets executed in interrupt context.